hw8/todolist/src/components/TagButtons/TagButtons.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from "react";
+import { tags, uniqueTagColors } from "../../App";
+import { Link, useLocation } from "react-router-dom";
+
+interface Props {
+  newTag: string;
+  setNewTag: (newTag: string) => void;
+  isLink?: boolean;
+}
+const TagButtons = ({ newTag, setNewTag, isLink = false }: Props) => {
+  const [tagColors, setTagColors] = useState(["#FFF", "#FFF", "#FFF", "#FFF"]);
```

Comment:
It's better to use enums instead of magic strings. When you want to change it, you can do it in only one place. Also, you can understad what color you're using just reading the code